exercise_3/ARecord.hpp

```diff
@@ -0,0 +1,20 @@
+//
+// Created by denys on 20.07.19.
+//
+
+#pragma once
+
+
+#include "IRecord.hpp"
+#include <list>
+
+class ARecord : public IRecord {
```

Comment:
ARecord isn't the most fortunate name.
Maybe something different, a bit more verbose?

Reply:
It's abstract class co I did it by analogy to IRecord

Reply:
Yeah, but the name `ARecord` isn't much verbose :)
Wanted to have a bit different name, only.

Reply:
Do you have the code style page, that we should stick to? Is it better to name it like AbstractRecort? I think it's logical to have the Record in the class name - because it's abstraction of Record class.

Reply:
No, there's no specific coding style for our training.
However, you might want to check some commonly used - i.e. by the Google: https://google.github.io/styleguide/cppguide.html - check for naming convention(s).